Sources/SnapshotTestingExtensions/SnapshotTestBase.swift

```diff
@@ -81,6 +86,9 @@ open class SnapshotTestBase: XCTestCase {
             let vc = UIHostingController(rootView: view)
             vc.overrideUserInterfaceStyle = uiStyle
             
+            // called to trigger rendering e.g. for AsyncImage 
+            vc.viewDidAppear(false)
```

Comment:
There are related open issues, seems like common problem.
https://github.com/pointfreeco/swift-snapshot-testing/issues/434

Some improvement advices
- Instead of statically call this, you can provide `UIHostingController` reference to the user in a closure and execute this closure before firing assertSnapshot method.
- Also you can set a default value to this closure so that you still would have this static call but this time user can modify it whenever needed.

```
func foo<Content: View>(
      _ update: @escaping (UIHostingController<View>) -> Void  = { $0.viewDidAppear(false) }
) -> Void { 
     // Implementation details
     update(vc)
     assertSnapshot(...)
 }
```

Reply:
Yes, also is it possible to add the related open issue as a comment, so that we can follow it and revert our workaround.